### ISSUE TITLE
Mixing events

### ISSUE THREAD

**Issue by adamvas** [pusher/pusher-websocket-dotnet#109]
Hi,

I migrated from 1.2 to 2.0 followed what you wrote and it seems `events`  can be mixed.
I bind all my events - using private channel - and it ends up like just pick an event in random manner.

```
channel.Bind("my-event-1", ExecuteEvent1);
channel.Bind("my-event-2", ExecuteEvent2);
channel.Bind("my-event-3", ExecuteEvent3);

private void ExecuteEvent*(PusherEvent e)
{
 ...
}
```

Did I miss sth?

**Comment by elverkilde** [comment]
This is indeed a bug, thanks for reporting. We're working on a fix, probably best to downgrade to 1.2 for now.

**Comment by AdamVasPrezi** [comment]
Hi, 2.0.1 seems fine! Thanks @elverkilde
